notes.md

```diff
@@ -53,13 +53,13 @@ Thanks to the PixelCropXXX elements, it's possible to crop the image before bein
 
 PixelXXX (size of the coded image) -> PixelCropXXX (size of the image to keep) -> DisplayXXX (resized cropped image)
 
-# Matroska version indicators
+# Matroska versioning
 
-The EBML Header each Matroska file starts with contains two version number fields that inform a reading application about what to expect. These are `DocTypeVersion` and `DocTypeReadVersion`.
+The `EBML Header` of each Matroska file MUST contain the two version number fields that inform a reading application on what to expect. These are `DocTypeVersion` and `DocTypeReadVersion`.
```

Comment:
This update maintains the original meaning which is fine, but it isn't true that the header MUST contain the version fields, as if not then the default values would be used. Would prefer to say something like "The EBML Header of each Matroska document informs the reading application on what version of Matroska to expect...."